mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping

```diff
@@ -13,6 +16,10 @@ CLASS net/minecraft/class_572 net/minecraft/client/render/entity/model/BipedEnti
 	FIELD field_3401 rightArm Lnet/minecraft/class_630;
 	METHOD <init> (F)V
 		ARG 1 scale
+	METHOD method_2804 lerpAngle (FFF)F
+		ARG 1 from
+		ARG 2 to
+		ARG 3 position
```

Comment:
Imo a name like "old" may be more indicated that this is the value to be lerped.

Reply:
I also found `start` and `end` but I will go with `old` and `to` if you think so.

Reply:
so the third param is the value that's actually lerped?

Reply:
the third on is the lerp percentage. It's lerped between `from` and `to`